make/dev.mk

```diff
@@ -46,3 +46,8 @@ else
 	release-please release-pr \
 	  --repo-url $(GITHUB_ORG)/$(GITHUB_REPOSITORY) --token "$$(gh auth token)"
 endif
+
+.PHONY: .envrc.e2e
+.envrc.e2e:
+	gojq --yaml-input --raw-output '.variables | to_entries | map("\(.key)=\(.value|tostring)")|.[]' < test/e2e/config/caren.yaml | envsubst > .envrc.e2e
```

Comment:
```suggestion
	gojq --yaml-input --raw-output '.variables | to_entries | map("export \(.key)=\(.value|tostring)")|.[]' < test/e2e/config/caren.yaml | envsubst > .envrc.e2e
```

Reply:
when i dont have these exported they dont work for me

Reply:
Agreed. I've fixed that up as you suggested.